WordPress/gutenberg
# Cover block: adding background-size and background-repeat properties to the block CSS

## What?

Adding background-size and background-repeat properties to the block cover CSS when it is set to fixed background.

## Why?

To make the fixed background functionality work more consistently with all the image sizes.


## How?

Previously we were not applying a background-size property so the background image was using `auto`, the default value. The default value [scales the background image in the corresponding direction such that its intrinsic proportions are maintained.](https://developer.mozilla.org/en-US/docs/Web/CSS/background-size).


> [Intrinsic dimensions and proportions](https://developer.mozilla.org/en-US/docs/Web/CSS/background-size#intrinsic_dimensions_and_proportions)
The computation of values depends on the image's intrinsic dimensions (width and height) and intrinsic proportions (width-to-height ratio). These attributes are as follows:
A bitmap image (such as JPG) always has intrinsic dimensions and proportions.

## Testing Instructions
1. Create a new post
2. Add a cover block
3. Add a cover block with a large background image - [like this one](https://cloudup.com/c5Uwi07_2q1)
4. Toggle the fixed background option
5. Check if the results are the expected.

## Screenshots or screencast 

**Before:**
| Editor | frontend |
| --------- | ----------- |
| ![image](https://user-images.githubusercontent.com/1310626/179233961-fcbf2acc-cc73-458d-9714-3cbd04280951.png) | ![image](https://user-images.githubusercontent.com/1310626/179234005-14a0d523-2259-41d9-9cb4-d2a91f375a00.png) |








**After:**
| Editor | frontend |
| --------- | ----------- |
| ![image](https://user-images.githubusercontent.com/1310626/179234064-92652158-effb-497e-bb52-cfb14eb6afc7.png) | ![image](https://user-images.githubusercontent.com/1310626/179234091-db089ba9-3724-4dc1-9d11-4e4a1085c2a6.png) |
